src/control/lib/support/log_test.go

```diff
@@ -29,9 +29,17 @@ func startFakeEngine(t *testing.T) bool {
 	cmd := exec.Command("bash", "-c", runCmd)
 	err := cmd.Start()
 	if err != nil {
+		t.Logf("FakeEngine Failed to start with err = %s", err.Error())
```

Comment:
A few items to clean up, while you are refactoring:

- Call `t.Helper()` at the start of the `startFakeEngine` and `killFakeEngine` helper functions. It makes debugging failures a bit easier.
- Call `t.Fatal()` with the log message for the error cases. Right now I don't think any of these will be detected, as the return value isn't checked.
- Remove the boolean return values, as they aren't checked anyway.

Reply:
Done